app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,441 @@
+  <string name="btn_close">Zavřít</string>
+
+  <!-- //////////////// settings //////////////// -->
+
+  <string name="settings_general_title">Obecné</string>
+  <string name="settings_display_prefs">Zobrazení</string>
+  <string name="settings_security_title">Bezpečnost</string>
+  <string name="settings_display_seed">Obnovovací fráze</string>
+  <string name="settings_seed_security">Nastavení přístupu</string>
+  <string name="settings_advanced_title">Pokročilé</string>
+  <string name="settings_fees">Nastavení poplatků</string>
+  <string name="settings_list_channels">Seznam kanálů</string>
+  <string name="settings_logs">Logy</string>
+  <string name="settings_electrum">Electrum server</string>
+  <string name="settings_mutual_close">Vynucené uzavření kanálů</string>
+  <string name="settings_force_close">Nebezpečná zóna</string>
+
+  <!-- //////////////// notification - watcher //////////////// -->
+
+  <string name="notif_watcher_cheating_title">Prosím, zapněte Phoenix</string>
+  <string name="notif_watcher_cheating_message">Byl zjištěn pokus o podvod.</string>
+
+  <string name="notif_watcher_connection_title">Phoenix potřebuje přístup k internetu</string>
+  <string name="notif_watcher_connection_message">Sledování na pozadí potřebuje přístup k internetu.</string>
+
+  <!-- //////////////// close all channels //////////////// -->
+
+  <string name="closechannels_title">Uzavřít kanály</string>
+  <string name="closechannels_checking_channels">Kontroluji kanály, prosím vyčkejte…</string>
+  <string name="closechannels_channels_recap">Právě teď máte %1$d Lightning kanálů s agregovaným zůstatkem %2$s.</string>
+  <string name="closechannels_channels_none">Právě nemáte žádné kanály, které by mohly být vyprázdněny.\n\nJděte do \'Seznam kanálů\' k zobrazení stavu vašich kanálů.</string>
+  <string name="closechannels_mutual_title">Vyprázdnit peněženku</string>
+  <string name="closechannels_mutual_instructions"><![CDATA[
+    Použijte tuto stránku k vyprázdnění vaší peněženky na Bitcoinovou adresu.
+  ]]></string>
+  <string name="closechannels_mutual_input_instructions">Finance mohou být odeslány do bitcoinové peněženky. Před odesláním se ujistěte se, že je adresa správná.</string>
+  <string name="closechannels_mutual_input_hint">Zadejte Bitcoinovou adresu</string>
+  <string name="closechannels_mutual_button">Vyprázdnit peněženku</string>
+  <string name="closechannels_force_title">Vynutit uzavření</string>
+  <string name="closechannels_force_instructions"><![CDATA[
+    Vynucené uzavření umožňuje jednostranné uzavření vašich kanálů.
+    <br/><br/>
+    Tato funkce není magické tlačítko na spravení všech problémů. Je zde jako bezpečností opatření a <b>mělo by být použito jen v extrémních případech</b>. Například pokud váš uzel (ACINQ) permanentně zmizí, což by vám zabránilo v utrácení vašich peněz. Ve všech ostatních případech <b>kontaktujte podporu, pokud nacházíte jakékoliv problémy</b>.
+    <br/><br/>
```

Comment:
Can you confirm that:

> váš uzel (ACINQ)

means: "your **peer** (ACINQ)" and not "your **node** (ACINQ)"? This message covers the case where your channel's counterpart disappears (and not your node).

Reply:
I am aware of this but Czech does not have proper translation for "peer" that would be better than this. I am open to suggestions. @elkimek ?

Reply:
Maybe we could use a circumlocution, like "the other side of the channel"?

Reply:
That works! Thanks